src/fsharp/infos.fs

```diff
@@ -845,7 +866,7 @@ type ILMethInfo =
 /// Describes an F# use of a method
 [<NoComparison; NoEquality>]
 type MethInfo = 
-    /// FSMeth(tcGlobals, declaringType, valRef, extensionMethodPriority).
+    /// FSMeth(tcGlobals, enclosingType, valRef, extensionMethodPriority).
```

Comment:
Commented out code.

Reply:
This one is part of the comment showing the use of the union case